controllers/org_rbac_controller.go

```diff
@@ -57,11 +57,13 @@ func (r *OrganizationRBACReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	}
 
 	var errs []error
-	for rb, cr := range r.DefaultClusterRoles {
-		if err := r.putRoleBinding(ctx, ns, rb, cr, org); err != nil {
-			l.WithValues("rolebinding", rb).Error(err, "unable to create rolebinding")
-			r.Recorder.Eventf(&ns, "Warning", "RoleBindingCreationFailed", "Failed to create rolebinding %q", rb)
-			errs = append(errs, err)
+	if _, nsNoAccess := ns.Labels["appuio.io/no-access"]; !nsNoAccess {
```

Comment:
Two nitpicks here:

1. I'd suggest to do an early return here instead of introducing more nesting. So basically:

``` golang
if _, nsNoAccess := ns.Labels["appuio.io/no-access"]; nsNoAccess {
    return ctrl.Result{}, nil
}
```

2. I'd check if the value in the label is true. Not just if it exists.  If I set `appuio.io/no-access: false` I'd expect the RBAC rules to be added

Reply:
I'd prefer having the label key as a constant (cf. `LabelRoleBindingUninitialized`) in the code, rather than a hard-coded string here.

Reply:
> Two nitpicks here:
> 
>     1. I'd suggest to do an early return here instead of introducing more nesting. So basically:
> 
> 
> ```go
> if _, nsNoAccess := ns.Labels["appuio.io/no-access"]; nsNoAccess {
>     return ctrl.Result{}, nil
> }
> ```

Having the early return would be more in line with how controller code is usually structured.

> 
>     2. I'd check if the value in the label is true. Not just if it exists.  If I set `appuio.io/no-access: false` I'd expect the RBAC rules to be added

I'd much prefer using the label value to determine the access to the namespace. As mentioned, I'd expect that `appuio.io/no-access: false` would still create the default RBAC.

Reply:
I refactored the code a bit to explicitely require a `true` value for the `appuio.io/no-access` label as well as do a early return